socketio-client.gemspec

```
@@ -19,5 +19,6 @@ Gem::Specification.new do |s|
 
   # specify any dependencies here; for example:
   # s.add_development_dependency "rspec"
-  s.add_dependency 'rest-client', '~> 1.8.0'
+  s.add_dependency 'rest-client', '~> 2.0.1'
```

```suggestion
  s.add_dependency 'rest-client', '>= 1.8.0'
```

Rather than restrict rest-client to 2.0+, we should *allow* 2.0, but still continue to support the 1.8 series. This provides a smoother transition path for consumers of socketio-client.